libs/php.json

```
@@ -19,6 +19,7 @@
   {"id": "php.retrofit-php", "imports": ["Tebru\\Retrofit"], "name": "Retrofit PHP", "repo": "tebru/retrofit-php", "tags": ["rest-client"], "tech": ["internet"]},
   {"id": "php.faker", "imports": ["Faker"], "name": "Faker", "repo": "fzaninotto/Faker", "tags": ["fake-data"], "tech": ["eng-tools"]},
   {"id": "php.phalcon", "imports": ["Phalcon"], "name": "Phalcon", "repo": "phalcon/cphalcon", "tags": ["framework"], "tech": ["php-web"]},
+  {"id": "php.php-cs-fixer", "imports": ["PhpCsFixer"], "name": "PHP CS Fixer", "repo": "FriendsOfPHP/PHP-CS-Fixer", "tags": ["cs", "coding-standards", "linter", "fixer"], "tech": ["eng-tools", "cli", "utils"]},
```

Since we currently show only one tag for a library in profiles, only "cs" will be visible. Could you move "coding-standards" to be the first?

```suggestion
  {"id": "php.php-cs-fixer", "imports": ["PhpCsFixer"], "name": "PHP CS Fixer", "repo": "FriendsOfPHP/PHP-CS-Fixer", "tags": ["coding-standards", "linter", "fixer"], "tech": ["eng-tools", "cli", "utils"]},
```